### PR TITLE
new split inbox endpoints for mobile

### PR DESCRIPTION
**Pull request by stkem**
@jaredpetker @thassss @DerekBurch @mrbrown14 
cc @andrewconner

### CODE REVIEW

File: kifi-backend/modules/eliza/conf/elizaService.routes [hunk 1]
```diff
@@ -41,6 +40,13 @@ GET     /m/1/eliza/clearMessageSearchHistory  @com.keepit.eliza.controllers.mobi
 POST    /eliza/devices/:deviceType        @com.keepit.eliza.controllers.mobile.MobileDevicesController.registerDevice(deviceType:String)
 
 ##########################################
+# Mobile V2
+##########################################
+GET     /m/2/eliza/thread/:threadId        @com.keepit.eliza.controllers.mobile.MobileMessagingController.getPagedThread(threadId: String, pageSize: Int ?= 1000, fromMessageId: Option[String] ?= None)
+GET     /m/2/eliza/notifications/messages  @com.keepit.eliza.controllers.mobile.MobileMessagingController.getMessageNotifications(n: Int, before: Option[String] ?= None)
+GET     /m/2/eliza/notifications/system    @com.keepit.eliza.controllers.mobile.MobileMessagingController.getSystemNotifications(n: Int, before: Option[String] ?= None)
+
```
Comment (stkem) [inline kifi-backend/modules/eliza/conf/elizaService.routes:48]: @jaredpetker @thassss @DerekBurch @mrbrown14

Comment (stkem) [inline kifi-backend/modules/eliza/conf/elizaService.routes:48]: good?

**Comment by stkem** [comment]
API ok @mrbrown14 @jaredpetker? Would like to get this into prod asap so we can fully test it.

**Comment by jaredpetker** [comment]
Checking it out now

File: kifi-backend/modules/eliza/conf/elizaService.routes (hunk 1 above)
Comment (jaredpetker) [inline kifi-backend/modules/eliza/conf/elizaService.routes:48]: lgtm

**Comment by jaredpetker** [comment]
looks great, thanks!

**Pull request merged**